BangazonAPI/BangazonAPI/Models/Department.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Department
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public int Budget { get; set; }
+
```

Comment:
From ticket 7: 

`If the query string parameter of ?_include=employees is provided, then all employees in the department(s) should be included in the response.`

Let's add a list of employees on here!